src/components/mgt-people/mgt-people.ts

```diff
@@ -11,10 +11,32 @@ import { Providers } from '../../Providers';
 import { ProviderState } from '../../providers/IProvider';
 import '../../styles/fabric-icon-font';
 import '../mgt-person/mgt-person';
-import { PersonCardInteraction } from '../mgt-person/mgt-person';
 import { MgtTemplatedComponent } from '../templatedComponent';
 import { styles } from './mgt-people-css';
 
+/**
+ * Defines how a person card is shown when a user interacts with
+ * a person component
+ *
+ * @export
+ * @enum {number}
+ */
+export enum PersonCardInteraction {
```

Comment:
Should use the same enum from the Person card, do not create a new one. I recommend moving to a new file (`PersonCardInteraction.ts`)

Reply:
Refactored Person Card Interaction to `src/components/PersonCardInteraction.ts`:
7defefd

Affects:
*mgt-people (also default switched to hover) 
*mgt-person-card
*mgt-person